pkg/oc/cli/cmd/newapp.go

```diff
@@ -723,7 +723,8 @@ func handleError(err error, baseName, commandName, commandPath string, config *n
 		// this print serves as a header for the printing of the errorGroups, but
 		// only print it if we precede with classification errors, to help distinguish
 		// between the two
-		fmt.Fprintln(buf, "Errors occurred during resource creation:")
+		fmt.Fprintf(buf, "Errors occurred during %s processing of arguments:", commandName)
+		fmt.Fprint(buf, "\n")
```

Comment:
why not just put this in the Fprintf above?

Reply:
will do 

also, an fyi, this path is proving rare now, in that most args pass the `IsComponentReference` test